src/05-derivatives/04-functional-derivatives.md

```diff
@@ -0,0 +1,373 @@
+        func/
+            <source_keywords>[_desc-<label>]_<mixing|components>.tsv
+            <source_keywords>[_desc-<label>]_<mixing|components>.nii.gz
+            <source_keywords>[_desc-<label>]_decomposition.json
+```
+
+for example:
+
+```Text
+sub-001/
+    func/
+        sub-001_task-rest_run-1_desc-MELODIC_components.nii.gz
+        sub-001_task-rest_run-1_desc-MELODIC_mixing.tsv
+        sub-001_task-rest_run-1_desc-MELODIC_decomposition.json
+```
+
+```Text
+sub-001/
+    func/
+        sub-001_task-rest_run-1_desc-tICA_mixing.nii.gz
+        sub-001_task-rest_run-1_desc-tICA_components.tsv
+        sub-001_task-rest_run-1_desc-tICA_decomposition.json
+```
+
+Spatiotemporal decompositions produce either spatial or temporal components,
+along with conjugate mixing matrices that can be represented as time series and
+spatial maps, respectively. The combination of suffix and extension indicates
+which class of algorithm produced the outputs.
+
+Mixing matrices may be omitted for algorithms that are temporal decompositions
+with no spatial component, such as CompCor variants.
+
+Spatiotemporal decomposition files MUST be accompanied by a data dictionary in
+JSON format, consistent with Raw-BIDS section 4.2, which describes metadata for
+each column name. In addition to column names, the following MANDATORY field is
+added:
+
+| Key name | Description                                           |
+| -------- | ----------------------------------------------------- |
+| Method   | REQUIRED. Algorithm name and (if applicable) version. |
+
+#### Column names
+
+Column names in spatiotemporal decompositions take the form of
+`<decomposition>_<index>`, where `<decomposition>` is the name of the
+decomposition algorithm, and `<index>` is a numeric identifier. Indices SHOULD
+start at 0. If there is a natural ordering, indices should reflect that
+ordering. For example, the aCompCor component which explains the most variance
+should be named `a_comp_cor_00`.
+
+The following reserved words indicate common algorithms:
+
+| Column name              | Description                                                                                                                                                                                                |
+| ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `[a|t|w|c]_comp_cor_<x>` | CompCor (Behzadi, et al., 2007) calculated with voxels chosen based on: `a`: anatomically derived ROIs (white matter and CSF), `t`: temporal variance, `w`: white matter voxels only, `c`: CSF voxels only |
+| `melodic_<x>`            | Columns from the mixing matrix in FSL MELODIC                                                                                                                                                              |
```

Comment:
This may be breaking the rule of adding features, but this recently came up that fMRIPrep uses `aroma_motion_<x>` to indicate a MELODIC time series identified as motion-related by the AROMA classifier. Should that go here? Or should this be a discussion for another day, and allow this to be a "non-standard" feature in fMRIPrep for now?

Reply:
This will impact tedana, too, but in my mind I was leaving that out of this draft as "non-standard"....

Reply:
yup - lets leave it to another PR